packages/iov-bns/src/decodeobjects.ts

```diff
@@ -170,6 +170,7 @@ export function decodeDomain(prefix: IovBech32Prefix, domain: codecImpl.account.
     hasSuperuser: ensure(domain.hasSuperuser, "hasSuperuser"),
     msgFees: ensure(domain.msgFees, "msgFees").map(decodeAccountMsgFee),
     accountRenew: asIntegerNumber(ensure(domain.accountRenew, "accountRenew")),
+    broker: domain.broker ? encodeBnsAddress(prefix, domain.broker) : ("" as Address),
```

Comment:
@albertandrejev, is `("" as Address)` the right value for the case when there's no broker?  If not then what's the right value?

Reply:
I believe so, the Domain interface does not let broker be optional. The only concern is what bnsd does with and empty string broker but I think it's correct.